src/clr-angular/utils/popover/providers/popover-position.service.ts

```diff
@@ -5,32 +5,42 @@
  *
  */
 import { isPlatformBrowser } from '@angular/common';
-import { Injectable, PLATFORM_ID, Inject } from '@angular/core';
+import { Injectable, PLATFORM_ID, Inject, EventEmitter } from '@angular/core';
 
 import { ClrPopoverEventsService } from './popover-events.service';
 import { ClrPopoverPosition } from '../interfaces/popover-position.interface';
 import { ClrPopoverContentOffset } from '../interfaces/popover-content-offset.interface';
 import { ClrViewportViolation } from '../enums/viewport-violation.enum';
 import { align, flipSidesAndNudgeContent, flipSides, nudgeContent, testVisibility } from '../position-operators';
 import { ClrAxis } from '../enums/axis.enum';
+import { Observable } from 'rxjs';
 
 @Injectable()
 export class ClrPopoverPositionService {
+  private _smartPositionReady: EventEmitter<ClrPopoverPosition> = new EventEmitter();
+  smartPositionReady: Observable<ClrPopoverPosition> = this._smartPositionReady.asObservable();
```

Comment:
Can we call this just smartPosition and drop the ready?

Reply:
Okay, we talked with Bozhidar, that we drop the current smartPosition property and only leave the Observable, but it will have to become BehaviorSubject then.